src/tests/dispatch-payload.test.ts

```diff
@@ -12,33 +12,32 @@ import {
   generateViewSubmissionPayload,
 } from "./test_utils.ts";
 
+const noop = () => "";
+
 Deno.test("DispatchPayload function", async (t) => {
   await t.step("should be defined", () => {
     assertExists(DispatchPayload);
   });
-  await t.step(
-    "should throw if an unrecognized event type is dispatched",
```

Comment:
Don't need this test as the unhandledEvent handler tests at the bottom of this file supersede this test case.